certified-connectors/AIForged/apiProperties.json

```diff
@@ -0,0 +1,66 @@
+{
+  "publisher": "Larc AI (PTY) Ltd",
+  "stackOwner": "Larc AI (PTY) Ltd",
+  "contact": {
+    "name": "AIForged",
+    "url": "https://www.aiforged.com",
+    "email": "info@aiforged.com"
+  },
+  "iconBrandColor": "#0E76BC",
+  "capabilities": [
+    "actions"
+  ],
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "AIForged",
+          "clientId": "swaggerui",
+          "clientSecret": "read the docs",
```

Comment:
@aiforged It is not requried to add in the apiPropertis.json file. Have you validated your connector artifacts using our [paconn](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/tools/paconn-cli) tool ?

Reply:
yes we did - here is the response without errors
<img width="888" alt="image" src="https://user-images.githubusercontent.com/25734528/149189478-c2a7bdb7-0398-49f1-a424-944e9d28e7e2.png">